### ISSUE TITLE
SR_RIDEINLIGHTNING Level Hit and SP

### ISSUE THREAD

**Issue by Badarosk0** [rathena/rathena#6386]
* **rAthena Hash**: https://github.com/rathena/rathena/commit/3d4021bfc61be0319aed26e5b496b80faa0e5f88

* **Client Date**: 2020-09-02

* **Server Mode**: RE

* **Description of Issue**: 
  * Result: 
Regardless of the level used in the skill, it is always generating 2 hits.

  
  * Expected Result: 
SR_RIDEINLIGHTNING skill must always consume 2 spheres, however it must generate different hits depending on the skill level. If the skill is used at level 01 then it should generate 1 hit. If the skill is used at level 5 it should generate 5 hits.

  
  * How to Reproduce: 

  * Official Information: 
  
  
```
	[SKID.SR_RIDEINLIGHTNING] = {
		"뇌광탄(雷光彈)",
		"MAX Lv : 5",
		"^777777습득조건 :  탄지신통 3 ^000000",
		"계열 : ^777777  액티브 / 데미지 ^000000",
		"내용 : ^777777  일정범위에 기 구체 2개를 발사, 폭발시켜 그 주변의 모든 대상에게 원거리 물리 데미지를 준다.",
		"시전자의 베이스 레벨에 따라 데미지가 추가로 증가되며, 시전자의 무기가 너클 계열일 경우 추가 데미지가 적용된다.^000000",
		"[Level 1] : ^777777공격력  40%/  90%(너클)/범위 3 x 3/1회 공격^000000",
		"[Level 2] : ^777777공격력  80%/ 180%(너클)/범위 3 x 3/2회 공격^000000",
		"[Level 3] : ^777777공격력 120%/270%(너클)/범위 5 x 5/3회 공격^000000",
		"[Level 4] : ^777777공격력 160%/360%(너클)/범위 5 x 5/4회 공격^000000",
		"[Level 5] : ^777777공격력 200%/450%(너클)/범위 7 x 7/5회 공격^000000"
	},
	
Also check SP consumption.	
	[SKID.SR_RIDEINLIGHTNING] = {
		"SR_RIDEINLIGHTNING",
		SkillName = "³ú±¤Åº(ÒâÎÃ÷¥)",
		MaxLv = 5,
		SpAmount = { 25, 30, 35, 40, 45 },
		bSeperateLv = true,
		AttackRange = { 11, 11, 11, 11, 11 },
		_NeedSkillList = {
			{ SKID.MO_FINGEROFFENSIVE, 3 }
		}
	},
```

* **Modifications that may affect results**:

**Comment by Balferian** [comment]
Do you apply this fix?
https://github.com/rathena/rathena/commit/77b103192c41a81a1a179723b7e9afd234b8e85e

**Comment by Badarosk0** [comment]
> Do you apply this fix? (for hits count) [77b1031](https://github.com/rathena/rathena/commit/77b103192c41a81a1a179723b7e9afd234b8e85e)

Yes, that's it. Had not seen, solves the problem. Thanks.
I think that only SP consumption is missing.
